lib/server/routes/buckets.js

```diff
@@ -1204,6 +1206,19 @@ BucketsRouter.prototype.removeFile = function (req, res, next) {
                 if (err) {
                   log.error('Error deleting pointer %s, reason: %s', pointer._id, err.message);
                 }
+                if (!err) {
```

Comment:
`err` nullability has been checked just above, using a `return` there or using an `else` here, could be more readable

Reply:
sorry about this one! totally right.

Fixed.

Also added nextPointer() instead of return when there's an error enqueuing a delete shard task.

Although unsure if this is the desired behavior (?)

If there's an error with the queue do we want to continue removing pointers from mongo or do we want to stop?